weasyl/ignoreuser.py

```diff
@@ -28,22 +28,19 @@ def cached_list_ignoring(userid):
                      [userid], options=["within"])
 
 
-def select(userid, limit, backid=None, nextid=None):
-    statement = ["SELECT iu.otherid, pr.username, pr.config FROM ignoreuser iu"
-                 " INNER JOIN profile pr ON iu.otherid = pr.userid"
-                 " WHERE iu.userid = %i" % (userid,)]
-
-    if backid:
-        statement.append(" AND pr.username < (SELECT username FROM profile WHERE userid = %i)" % (backid,))
-    elif nextid:
-        statement.append(" AND pr.username > (SELECT username FROM profile WHERE userid = %i)" % (nextid,))
-
-    statement.append(" ORDER BY pr.username%s LIMIT %i" % (" DESC" if nextid else "", limit))
+def select(userid):
+    results = d.engine.execute(
```

Comment:
More of a stylistic choice here, but don't we typically use the ``"""triple quoted strings"""`` for SQL statements?
```suggestion
    results = d.engine.execute("""
        SELECT iu.otherid, pr.username FROM ignoreuser iu
          INNER JOIN profile pr ON iu.otherid = pr.userid
        WHERE iu.userid = %(user)s
        ORDER BY lower(pr.username)
    """, user=userid)
```

Reply:
There’s a lot of both in the codebase. I picked this one to include less unnecessary whitespace in the string, because it’s started to bother me that the indents are inside it :P

Reply:
Fair enough; LGTM, otherwise.